### ISSUE TITLE
Latest release breaks test

### ISSUE THREAD

**Issue by zemirco** [standard/standard#969]
I've got a simple React component:

```js
export default class MyComponent extends React.Component {
  state = {
    active: false
  }

  render () {
    return (
      <div />
    )
  }
}
```

It always worked as expected but suddenly my tests at Travis are failing

```
'state' is not defined
```

Do I have to change anything in my config for the latest release? Here is the important part from my `package.json`. I'm using version `"standard": "^10.0.2"`

```json
  "standard": {
    "parser": "babel-eslint"
  }
```

After removing `node_modules` and reinstalling everything I also get the error on my local machine.

**Comment by bcomnes** [comment]
Is your babel set up to handle class properties?  Thats the first thing that comes to mind.

**Comment by zemirco** [comment]
Here is our `.babelrc`

```
{
  "env": {
    "test": {
      "presets": [
        ["env", {
          "modules": "commonjs"
        }]
      ],
      "plugins": ["transform-class-properties"]
    }
  },
  "presets": [
    "react",
    ["env", {
      "targets": {
        "browsers": ["last 2 versions", "safari >= 9"]
      },
      "useBuiltIns": true,
      "debug": true
    }]
  ],
  "plugins": ["transform-object-rest-spread", "transform-class-properties"]
}
```

Did you change anything with the latest release? It was working fine before.

**Comment by bcomnes** [comment]
We updated to the latest standard JSX plugin to fix some other reported issues https://github.com/standard/standard/commit/d22676e6cee35965d9b9231cd9e6c93588cacd09 which had a few minor changes https://github.com/standard/eslint-config-standard-jsx/commits/master.

**Comment by vpcvdc** [comment]
### Workaround solution
Remove parser property from standard object in package.json file
```
{
  "standard": {
    "parser": "babel-eslint"
  }
}
```
Add parser option in command line, like this
`standard --parser babel-eslint`

**Comment by Secretmapper** [comment]
@vpcvdc's workaround does not work for me.

This is a pretty big showstopper for us as currently we have a lot of fails in Travis + force committing (standard is part of my precommit hook)

**Comment by gillesdemey** [comment]
Duplicate of https://github.com/standard/standard/issues/944?

**Comment by feross** [comment]
I think I just ran into this too. Started happening when I installed another package that had `eslint@4` as a dependency alongside `standard`. In my case it was `@sonarwhal/sonar`.

Looks like `standard` is somehow using the wrong eslint version (we're still on `eslint@3` as of `standard@10.0.2`) Removing that package solved the problem.

You can find other packages using eslint by running `npm ls eslint` in your project folder.

**Comment by Flet** [comment]
I believe this will be fixed in the next version of `standard` where `eslint` and all plugins are bumped to the latest version.

Can you give it a try? `npm install standard@beta`

**Comment by xcv58** [comment]
I verified this is fixed by `standard@11.0.0-beta.0`, details:
https://github.com/xcv58/standard-issue-969/commit/1ce51e30f38a0f317dc51b992794691147dd1cda

**Comment by feross** [comment]
🗣 Standard 11 is released! Run `npm install standard@latest --save-dev` to update to the latest version. This will also update the version in `package.json` ✨

Changelog: https://github.com/standard/standard/blob/master/CHANGELOG.md#1100---2018-02-18
